src/repositories/user.repository.ts

```diff
@@ -41,4 +42,23 @@ export const addUserData = async (user: User) => {
   }
 };
 
+export const findUser = async (user: User) => {
+  const { username, password } = user;
+
+  if (user && username && password) {
```

Comment:
wrong condition, because if `user` doesn't exist, then you'll have error on line 46